pom.xml

```diff
@@ -97,6 +97,16 @@
         <pax.cdi.version>0.12.0</pax.cdi.version>
         <validation.version>1.1.0.Final</validation.version>
         <commons-configuration.version>1.10</commons-configuration.version>
+        <servicemix.bundles.mina.version>1.1.7_6</servicemix.bundles.mina.version>
```

Comment:
All of these version variables are in the copy/pasted features file and are being used. I don't see these version variables becoming stale unless we upgrade to newer a version of camel that doesn't use a particular dependency that we have defined a version variable for. In that case, we just have to remember to remove it.